src/components/InputBlocks/GoogleUploadInputBlock/GoogleUploadInputBlock.jsx

```diff
@@ -0,0 +1,129 @@
+// REACT LIBS
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import GoogleButton from 'react-google-button';
+import GooglePicker from 'react-google-picker';
+
+// UI LIB COMPONENTS
+import { Upload } from 'antd';
+
+// COMPONENTS
+import { InputBlockContainer } from 'components';
+
+// GOOGLE CREDENTIALS
+const CLIENT_ID =
+  '438861478238-uq13v0ho59r44dh9j5a05agg4ap3ujs7.apps.googleusercontent.com';
+const DEVELOPER_KEY = 'AIzaSyDAoTfHvUJO-Gent8_FmW1oDnvRg-8dXlI';
+const SCOPE = ['https://www.googleapis.com/auth/drive.readonly'];
```

Comment:
@shiguemi32 hmmm acho que vou solicitar a geração credenciais no helpdesk neste projeto:
https://console.cloud.google.com/apis/credentials?folder=&organizationId=&project=dtc-plataforma-ia

Você lembra em que telas que dá pra gerar o CLIENT_ID e DEVELOPER_KEY? E quais as informações que a gente precisa passar (tinha umas URLs lá né? não sei como fica no caso de uma plataforma que pode estar em qquer URL).

Reply:
Precisa ativar o Google Drive API:
https://console.developers.google.com/apis/library/drive.googleapis.com?q=drive&id=e44a1596-da14-427c-9b36-5eb6acce3775&project=argon-acumen-169500

Ir no console https://console.developers.google.com/apis/dashboard?project=argon-acumen-169500
Entrar em Credenciais e gerar uma Chave de API e uma ID do cliente OAuth 2.0

As URLs ficam no ID do cliente OAuth 2.0
Ai não sei como a gente faria no caso da plataforma
Acho que vai precisa fica adicionando as URL nessa chave...

Reply:
@shiguemi32 vou aprovar com esta key, depois que o chamado do helpdesk for atendido a gente troca,